### ISSUE TITLE
Package is not installable via composer

### ISSUE THREAD

**Issue by BorislavSabev** [FloeDesignTechnologies/phpcs-security-audit#10]
Hello.

Over at Packagist [there is a package of this repo](https://packagist.org/packages/pheromone/phpcs-security-audit). I don't know if the maintainers here created it  but it is not working installable.
You can install with Composer pointing to this repo of course.

If the package is not yours can you add a good package for this repo?
I can help with it.

**Comment by kenorb** [comment]
Error:

```
$ composer require pheromone/phpcs-security-audit
You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug

In InitCommand.php line 642:
                                                                                                                                                                  
  Could not find package pheromone/phpcs-security-audit at any version for your minimum-stability (stable). Check the package spelling or your minimum-stability  
```

**Comment by jmarcil** [comment]
```
$ composer require pheromone/phpcs-security-audit

Using version ^1.0 for pheromone/phpcs-security-audit
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 2 installs, 0 updates, 0 removals
  - Installing squizlabs/php_codesniffer (3.2.2): Downloading (100%)
  - Installing pheromone/phpcs-security-audit (1.0.0): Downloading (connecting..Downloading (100%)
Writing lock file
Generating autoload files
```
solved #9 and updated https://packagist.org/packages/pheromone/phpcs-security-audit fixed this.
